apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/WalletOverview/WalletOverview.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { FC, useState } from 'react';
+
+import { t } from 'i18next';
+
+import {
+  Button,
+  ErrorBadge,
+  ErrorLevel,
+  Paragraph,
+  ParagraphSize,
+  Tabs,
+  TabSize,
+  TabType,
+} from '@sovryn/ui';
+import { Decimal } from '@sovryn/utils';
+
+import { AmountRenderer } from '../../../../2_molecules/AmountRenderer/AmountRenderer';
+import { useWalletConnect } from '../../../../../hooks';
+import { translations } from '../../../../../locales/i18n';
+import { BorrowAction } from './components/BorrowAction/BorrowAction';
+import { SupplyAction } from './components/SupplyAction/SupplyAction';
+
+const pageTranslations = translations.aaveReserveOverviewPage;
+
+type WalletOverviewProps = {
+  asset: string;
+};
+
+export const WalletOverview: FC<WalletOverviewProps> = ({
+  asset: initialAsset,
+}) => {
+  const [asset, setAsset] = useState(initialAsset);
+  const { account, connectWallet, pending } = useWalletConnect();
+
+  const assetBalance = Decimal.from(0); // TODO: mocked
+  const maxSupplyCap = true; // TODO: mocked
+
+  return (
+    <div className="bg-gray-90 p-6 rounded space-y-6 lg:bg-gray-90 lg:p-6 border border-gray-60">
+      <Paragraph className="text-base">
+        {t(pageTranslations.yourWalletTab.title)}
+      </Paragraph>
+
+      {account ? (
+        <>
+          <div>
+            {['ETH', 'WETH'].includes(asset) && (
```

Comment:
are ETH and WETH mock values? If not, it could be better define them as constants

Reply:
Mmm no these are not mocks and the condition should remain. I believe there're some constants for symbols so I'll add that. Thanks!

Reply:
Added here https://github.com/wakeuplabs-io/sovryn-dapp/pull/11/files#diff-4f6a55753296270a3ac94fc90b386b817449cefaee514d298a124325507d3d78R26